apps/election-manager/src/screens/ManualDataImportScreen.tsx

```diff
@@ -0,0 +1,415 @@
+        contestTallies: newContestTallies,
+      },
+    })
+  }
+
+  let totalNumberBallotsEntered = 0
+  const enteredDataRows: ReactChild[] = []
+  for (const precinct of election.precincts) {
+    const tally = talliesByPrecinct[precinct.id] ?? getEmptyExternalTally()
+    enteredDataRows.push(
+      <tr>
+        <TD>{precinct.name}</TD>
+        <TD narrow nowrap>
+          {tally.numberOfBallotsCounted}
+        </TD>
+      </tr>
+    )
+    totalNumberBallotsEntered += tally.numberOfBallotsCounted
+  }
+  const currentContests = expandEitherNeitherContests(
+    getContestsForPrecinct(election, currentPrecinctId)
+  )
+
+  return (
+    <React.Fragment>
+      <NavigationScreen>
+        <Prose maxWidth={false}>
+          <h1>Manually Add External Results </h1>
+          <p>Select the voting method for these ballots.</p>
+          <SegmentedButton>
+            <Button
+              disabled={ballotType === VotingMethod.Precinct}
+              onPress={() => setBallotType(VotingMethod.Precinct)}
+            >
+              Precinct
+            </Button>
+            <Button
+              disabled={ballotType === VotingMethod.Absentee}
+              onPress={() => setBallotType(VotingMethod.Absentee)}
+            >
+              Absentee
+            </Button>
+          </SegmentedButton>
+          <p>Select precinct to enter results for:</p>
+          <Select
+            data-testid="selectPrecinct"
+            value={currentPrecinctId}
+            name="precinct"
+            onChange={(e) => setCurrentPrecinctId(e.currentTarget.value)}
+            onBlur={(e) => setCurrentPrecinctId(e.currentTarget.value)}
+          >
+            <option value="" disabled>
+              Precinct
+            </option>
+            {election.precincts.map((precinct) => (
+              <option key={precinct.id} value={precinct.id}>
+                {precinct.name}
+              </option>
+            ))}
+          </Select>
+          {currentPrecinct && (
+            <React.Fragment>
+              <h3>Contest Results for {currentPrecinct.name}</h3>
+              {currentContests.map((contest) => {
+                let contestTitle = contest.title
+                if (contest.partyId) {
+                  const party = election.parties.find(
+                    (p) => p.id === contest.partyId
+                  )
+                  if (party) {
+                    contestTitle = `${contestTitle} - ${party.fullName}`
+                  }
+                }
+                return (
+                  <ContestData key={contest.id}>
+                    {contestTitle} <br />
+                    Total Ballots Cast:{' '}
+                    <TallyInput
+                      name={`${contest.id}-numBallots`}
+                      value={getValueForInput(contest.id, 'numBallots')}
+                      onChange={(e) =>
+                        updateContestData(contest.id, 'numBallots', e)
+                      }
```

Comment:
Something about how this is set up means I can't actually delete the contents of the field. This is pretty confusing since I can't backspace over the `0` and type what I want.

I realize you can just select the contents or just start typing with the insertion point after the 0, but it's confusing.

Reply:
I don't have a great idea on how to fix this without adding a lot of extra and confusing logic to everything. Ultimately I agree its slightly less than the ideal situation but I think its ok given that you can just type a number and it will replace the 0, or I think the most common scenario for mass data entry is tabbing through the boxes which automatically selects the 0.

Reply:
I agree tabbing through is how we would do it, but a novice computer user will probably use the mouse. Right?

Reply:
I agree that we will likely see users using the mouse & trying to delete a 0. 

I know I'm a crazy raw HTML old fart here, but can we not simply wait to validate until pressing "Save"?